frontend/src/scenes/persons/PersonDisplay.tsx

```diff
@@ -119,8 +119,8 @@ export function PersonDisplay({
                 }
                 visible={visible}
                 onClickOutside={() => setVisible(false)}
-                placement="right"
-                fallbackPlacements={['bottom', 'top']}
+                placement="top"
+                fallbackPlacements={['bottom', 'right']}
```

Comment:
@benjackwhite ,you were the last person to touch this area of the product.  I checked out other data tables in the UI where we display person data and my changes looked safe, but any risks from your perspective about changing this?

Reply:
nope sounds good